test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1605,62 +1628,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 	)
 })
 
-func setupPaddingPodLevel(fxt *e2efixture.Fixture, nrtList nrtv1alpha1.NodeResourceTopologyList, padInfo paddingInfo) []*corev1.Pod {
-	baseload, err := nodes.GetLoad(fxt.K8sClient, padInfo.targetNodeName)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", padInfo.targetNodeName)
-	By(fmt.Sprintf("computed base load: %s", baseload))
-
-	By(fmt.Sprintf("preparing target node %q to fit the test case", padInfo.targetNodeName))
-	// first, let's make sure that ONLY the required res can fit in either zone on the target node
-	nrtInfo, err := e2enrt.FindFromList(nrtList.Items, padInfo.targetNodeName)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing NRT info for %q", padInfo.targetNodeName)
-
-	// if we get this far we can now depend on the fact that len(nrt.Zones) == len(pod.Spec.Containers) == 2
-
-	paddingPods := []*corev1.Pod{}
-
-	// as low as you can get, that's why it's hardcoded
-	targetUnsuitableRes := corev1.ResourceList{
-		corev1.ResourceCPU:    resource.MustParse("2"),
-		corev1.ResourceMemory: resource.MustParse("2Gi"),
-	}
-
-	var zone nrtv1alpha1.Zone
-	// fix target node
-	zone = nrtInfo.Zones[0]
-	By(fmt.Sprintf("padding node %q zone %q to fit only %s", nrtInfo.Name, zone.Name, e2ereslist.ToString(targetUnsuitableRes)))
-	padPod, err := makePaddingPod(fxt.Namespace.Name, "target", zone, targetUnsuitableRes)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-
-	padPod, err = pinPodTo(padPod, nrtInfo.Name, zone.Name)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-
-	err = fxt.Client.Create(context.TODO(), padPod)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-	paddingPods = append(paddingPods, padPod)
-
-	podTotRes := e2ereslist.FromGuaranteedPod(*padInfo.pod)
-	By(fmt.Sprintf("testpod resource requests (vanilla): %s", e2ereslist.ToString(podTotRes)))
-	baseload.Apply(podTotRes)
-	By(fmt.Sprintf("testpod resource requests (adjusted): %s", e2ereslist.ToString(podTotRes)))
-
-	zone = nrtInfo.Zones[1]
-	By(fmt.Sprintf("padding node %q zone %q to fit only %s", nrtInfo.Name, zone.Name, e2ereslist.ToString(podTotRes)))
-	padPod, err = makePaddingPod(fxt.Namespace.Name, "target", zone, podTotRes)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-
-	padPod, err = pinPodTo(padPod, nrtInfo.Name, zone.Name)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-
-	err = fxt.Client.Create(context.TODO(), padPod)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred())
-	paddingPods = append(paddingPods, padPod)
-
-	paddingPodsUnsuitable := setupPaddingForUnsuitableNodes(2, fxt, nrtList, padInfo)
-	return append(paddingPods, paddingPodsUnsuitable...)
-}
-
-func setupPaddingContainerLevel(fxt *e2efixture.Fixture, nrtList nrtv1alpha1.NodeResourceTopologyList, padInfo paddingInfo) []*corev1.Pod {
+func setupPaddingLevel(fxt *e2efixture.Fixture, nrtList nrtv1alpha1.NodeResourceTopologyList, padInfo paddingInfo) []*corev1.Pod {
```

Comment:
this makes sense also because at conceptual level padding for scope=container is a more complex implementation of the padding for scope=pod. IOW, we can reimplement the `setupPaddingPodLevel` using the righe combination of parameters for `setupPaddingContainerLevel` (which you are doing here), while the opposite is not and can't be true.

I like this simplification, but I'd like to see this running for a while. The best way is actually get it running for a while and pay close attention to the test runs. Let's be vigilant for the time being.

While we are at it, let's rename the function `setupPadding` (minor, but still)